.github/workflows/ci.yml

```diff
@@ -139,13 +139,13 @@ jobs:
           set -e
 
           # build for Emscripten/WebGL
-          cargo clippy --target ${{ matrix.target }} -p wgpu -p wgpu-hal --no-default-features --features webgl,emscripten
+          cargo clippy --target ${{ matrix.target }} -p wgpu -p wgpu-hal --no-default-features
 
           # build cube example
-          cargo clippy --target ${{ matrix.target }} --example cube --features webgl,emscripten
+          cargo clippy --target ${{ matrix.target }} --example cube
 
           # build raw-gles example
-          cargo clippy --target ${{ matrix.target }} --example raw-gles --features webgl,emscripten
+          cargo clippy --target ${{ matrix.target }} --example raw-gles
```

Comment:
I had to double-check this again because I forgot, but basically the `webgl` crate feature does nothing for emscripten, it's enabled by default.
This was already in place in `Cargo.toml`, I just adjusted the `cfg`s to account for that.

I don't know anything about the emscripten backend, but my impression was that emscripten only works with the WebGL backend.

Reply:
Yeah this looks fine.